gauntlet/gauntlet.go

```diff
@@ -68,7 +68,8 @@ type ExecCommandOptions struct {
 }
 
 // ExecCommand Executes a gauntlet command with the provided arguments.
-//  It will also check for any errors you specify in the output via the errHandling slice.
+//
```

Comment:
```suggestion
```

Reply:
go fmt automatically adds this and if it isn't set like this then the linter gets angry.